Description: Terminate AM_TESTS_ENVIRONMENT by a semicolon.
 The automake manual says: "the contents of a non-empty AM_TESTS_ENVIRONMENT
 must be terminated by a semicolon." Without the semicolon, the perl command
 will indefinitely wait for input from stdin.
Index: tests/Makefile.am
===================================================================
--- tests/Makefile.am	(revision 12957)
+++ tests/Makefile.am	(working copy)
@@ -5,6 +5,6 @@
 TESTS_ENVIRONMENT = \
 	TWOLAME_CMD="$(top_builddir)/frontend/twolame" \
 	STWOLAME_CMD="$(top_builddir)/simplefrontend/stwolame" \
-	perl -w -Mstrict -MTest::Harness -e "runtests(@ARGV)"
+	perl -w -Mstrict -MTest::Harness -e "runtests(@ARGV)";
 
 CLEANFILES = *.mp2 *.raw
